js/preloader.js

```diff
@@ -1,147 +1,178 @@
-        delay = 10;
-      }
+    if (count > 8) {
+      delay = 50;
     }
     
-    if (count < strings.length) {
-      setTimeout(function() {
-        return addLog();
-      }, delay);
-    } else {
-      setTimeout(function() {
-        delay = 1000;
-        return createLog("ok");
-      }, 1000);
+    if (count > 10) {
+      delay = 10;
+    }
+  } else {
+    if (count > 3) {
+      delay = 10;
     }
   }
   
-  function createLog(type, index) {
-    var row = document.createElement('div');
-    
-    var spanStatus = document.createElement('span');
-    spanStatus.className = type;
-    spanStatus.innerHTML = type.toUpperCase();
-    
-    var message = (index != null) 
-                ? strings[index] 
-                : 'kernel: Initializing...';
-
-    if(index == null) 
-    {
-      var preloader = $('#preloader');
-      jQuery(preloader).fadeOut("slow");
-      jQuery("#main").fadeIn("slow");
-    }
-    
-    var spanMessage = document.createElement('span');
-    spanMessage.innerHTML = message;
-    
-    row.appendChild(spanStatus);
-    row.appendChild(spanMessage);
-    
-    return row;
+  if (count < strings.length) {
+    setTimeout(function() {
+      return addLog();
+    }, delay);
+  } else {
+    setTimeout(function() {
+      delay = 1000;
+      return createLog("ok");
+    }, 1000);
   }
+}
+
+function createLog(type, index) {
+  var row = document.createElement('div');
+  
+  var spanStatus = document.createElement('span');
+  spanStatus.className = type;
+  spanStatus.innerHTML = type.toUpperCase();
   
-  function goScrollToBottom() {
-    $(document).scrollTop($(document).height()); 
+  var message = (index != null) 
+              ? strings[index] 
+              : 'kernel: Initializing...';
+
+  if(index == null) 
+  {
+    var preloader = $('#preloader');
+    jQuery(preloader).fadeOut("slow");
+    jQuery("#main").fadeIn("slow");
   }
   
-  addLog();
+  var spanMessage = document.createElement('span');
+  spanMessage.innerHTML = message;
+  
+  row.appendChild(spanStatus);
+  row.appendChild(spanMessage);
   
-  
\ No newline at end of file
+  return row;
+}
+
+function goScrollToBottom() {
+  $(document).scrollTop($(document).height()); 
+}
+
+function setCookie(cname,cvalue,exdays) {
+  var d = new Date();
+  d.setTime(d.getTime() + (exdays*24*60*60*1000));
+  var expires = "expires=" + d.toGMTString();
+  document.cookie = cname + "=" + cvalue + ";" + expires + ";path=/";
+}
+function getCookie(cname) {
+  var name = cname + "=";
+  var decodedCookie = decodeURIComponent(document.cookie);
+  var ca = decodedCookie.split(';');
+  for(var i = 0; i < ca.length; i++) {
+    var c = ca[i];
+    while (c.charAt(0) == ' ') {
+      c = c.substring(1);
+    }
+    if (c.indexOf(name) == 0) {
+      return c.substring(name.length, c.length);
+    }
+  }
+  return "";
+}
+function checkCookie() {
+  var user=getCookie("LAKSHYA#CTF#INC#2020");
+  if (user != "") {    
+    createLog("ok");
```

Comment:
Also, can you mention your thoughts on why you're calling createLog?

Reply:
@ashawe It was a mistake i left it while debugging. I have made changes and removed the createLog.
Please review the Changes.